app/models/evmbaseline.rb

```diff
@@ -17,4 +21,15 @@ def bac
     evmbaselineIssues.sum(:estimated_hours).round(1)
   end
 
+  acts_as_event :title => Proc.new { |o| l(:title_evm_tab) + ':' + o.subject },
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use `proc` instead of `Proc.new`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.